README.md

```diff
@@ -56,48 +73,79 @@ $ source setup/activate_serve.sh
+Most of the recent issues encountered have been due to incompatible setup. We
+have now:
+- locked down the dependencies,
+- created setup and teardown scripts to setup self-contained environments with
+  those dependencies, and
+- CI enabled to validate that they continue work.
 
-One advantage of using `skip` authentication in development mode is that any user email can be entered without a password. Developers can use one of the emails that they loaded test data for in step (3) above. So if the test data loaded was with `-u shankari@eecs.berkeley.edu`, then the login email for the phone app would also be `shankari@eecs.berkeley.edu`.
+If you have setup failures, please compare the configuration in the **passing CI
+builds** with your configuration. That is almost certainly the source of the error.
 
-Updating the e-mission-\* plugins or adding new plugins
----
-[![osx-build-ios](https://github.com/e-mission/e-mission-phone/actions/workflows/ios-build.yml/badge.svg)](https://github.com/e-mission/e-mission-phone/actions/workflows/ios-build.yml)
-[![osx-build-android](https://github.com/e-mission/e-mission-phone/actions/workflows/android-build.yml/badge.svg)](https://github.com/e-mission/e-mission-phone/actions/workflows/android-build.yml)
+### Tested on
+__MacOS__
+- Intel chip, MacOS Ventura 13.6
+- Intel chip, MacOS Ventura 13.5.2
+- Intel chip, MacOS Ventura 13.0
+- Intel chip, MacOS Monterey 12.6.7
 
 Pre-requisites
 ---
-- the version of xcode used by the CI
+- The version of xcode used by the CI.
     - to install a particular version, use [xcode-select](https://www.unix.com/man-page/OSX/1/xcode-select/)
     - or this [supposedly easier to use repo](https://github.com/xcpretty/xcode-install)
     - **NOTE**: the basic xcode install on Catalina was messed up for me due to a prior installation of command line tools. [These workarounds helped](https://github.com/nodejs/node-gyp/blob/master/macOS_Catalina.md).
 - git
-- Java 11. Tested with [OpenJDK 11 (Temurin) using AdoptOpenJDK](https://adoptopenjdk.net/releases.html).
-- android SDK; install manually or use setup script below. Note that you only need to run this once **per computer**.
+- Java 17. Tested with [OpenJDK 17 (Temurin) using AdoptOpenJDK](https://adoptium.net). 
+- if you are not on the most recent version of OSX: `homebrew`
+    - this allows us to install the current version of cocoapods without
+      running into ruby incompatibilities - e.g.
+      https://github.com/CocoaPods/CocoaPods/issues/11763
+
+__1. Export statements__
```

Comment:
This is inconsistent with the rest of our build process. We should not expect people to copy-and-paste a lot of commands, everything should be in the setup/active scripts so that it can be tested by the CI. This was originally not in the setup since the SDK would be installed while installing android studio. But now that we install the SDK manually, we should include this among the other environment variables that we define.